examples/update-demo/nautilus-rc.yaml

```diff
@@ -0,0 +1,21 @@
+kind: ReplicationController
+id: update-demo-nautilus
+apiVersion: v1beta1
+desiredState:
+  replicas: 2
+  replicaSelector:
+    name: update-demo
+    version: nautilus
+  podTemplate:
+    desiredState:
+      manifest:
+        containers:
+        - name: update-demo
+          image: DOCKER_HUB_USER/update-demo:nautilus
+          ports:
+          - hostPort: 8080
```

Comment:
I think hostPort is fine for examples. It's easy to understand and setup firewall rules for, and keeps this example simple (just creates rc, no need to also create a service with external loadbalancer).